docs/user-guide.md

```diff
@@ -539,7 +539,7 @@ spec:
   template:
     type: Scala
     mode: cluster
-    image: gcr.io/spark/spark:v2.4.4
+    image: gcr.io/spark/spark:v2.4.5-SNAPSHOT
```

Comment:
`v2.4.5-SNAPSHOT` -> `2.4.5-SNAPSHOT`

Reply:
And it shouldn't include the v for the snapshot build.